AlertaDengue/Makefile

```
@@ -0,0 +1,7 @@
+clean:
+	python setup.py clean
+	find ./ -name '*.pyc' -type f -delete
+  
+develop: clean
+	python setup.py develop
+	pre-commit install
```

talvez esse comando deveria referenciar um nível atrás: 
```suggestion
	pre-commit install ..
```

para testar antes de executar `make develop` remova o pre-commit hook:

```
rm .git/hooks/pre-commit
```